openpype/hosts/aftereffects/plugins/publish/collect_review.py

```diff
@@ -0,0 +1,32 @@
+"""
+Requires:
+    None
+
+Provides:
+    instance     -> family ("review")
+"""
+
+import os
+
+import pyblish.api
+
+from openpype.pipeline.create import get_subset_name
+
+
+class CollectReview(pyblish.api.InstancePlugin):
+    """Add review to families if instance created with 'mark_for_review' flag
+    """
+
+    label = "Collect Review"
+    label = "Review"
+    hosts = ["aftereffects"]
+    order = pyblish.api.CollectorOrder + 0.1
+
+    publish = True
+
+    def process(self, instance):
+        creator_attributes = instance.data.get("creator_attributes")
+        if (creator_attributes and
+                creator_attributes.get("mark_for_review") and
+                "review" not in instance.data["families"]):
+            instance.data["families"].append("review")
```

Comment:
Unused attributes `publish`, doubled `label` and the plugin does not have any instance filtering so can be changed to context plugin (faster processing).
```suggestion
class CollectReview(pyblish.api.ContextPlugin):
    """Add review to families if instance created with 'mark_for_review' flag
    """

    label = "Review"
    hosts = ["aftereffects"]
    order = pyblish.api.CollectorOrder + 0.1

    def process(self, context):
        for instance in context:
            creator_attributes = instance.data.get("creator_attributes") or {}
            if (
                creator_attributes.get("mark_for_review")
                and "review" not in instance.data["families"]
            ):
                instance.data["families"].append("review")
```